File: Python/Product.py
class Product():
	
	# atribut private	
	__price = ""
	__idProduct = ""

	# konstruktor
	def __init__(self):
		self.__price = ""
		self.__idProduct = ""

	# setter dan getter untuk atribut price
	def setPrice(self, price):
		self.__price = price

	def getPrice(self):
		return self.__price

	# setter dan getter untuk atribut idProduct
	def setIdProduct(self, idProduct):
		self.__idProduct = idProduct

	def getIdProduct(self):
		return self.__idProduct;

	# method untuk mencetak output atribut dari class Product
	def printProduct(self):
		print("Harga          : " + self.getPrice())
		print("ID Produk      : " + self.getIdProduct())

File: PHP/Product.php
<?php

class Product{
	private $price = "";
	private $idProduct = "";

	/* konstruktor */
	public function __construct($price = "", $idProduct = ""){
		$this->price = $price;
		$this->idProduct = $idProduct;
	}

	/* setter dan getter untuk atribut price */
	public function setPrice($price){
		$this->price = $price;
	}

	public function getPrice(){
		return $this->price;
	}

	/* setter dan getter untuk atribut idProduct */
	public function setIdProduct($idProduct){
		$this->idProduct = $idProduct;
	}

	public function getIdProduct(){
		return $this->idProduct;
	}

	/* method untuk mencetak output atribut dari class Product */
	public function printProduct(){
		echo "Harga          : ".$this->getPrice()."<br>";
		echo "ID Produk      : ".$this->getIdProduct()."<br>";
	}
}

?>